server/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../db/index');
+let app = express();
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.get('/rooms/similarListings', (req, res) => {
+  //
+  let roomId = req.body;
+  console.log('Req: ', roomId);
+  db.fetchSimilarListings(roomId, (err, listings) => {
+    if (err) {
+      res.status(400).send(`Unable to find similar listings for ${roomId}`);
+    } else {
+      res.status(200).send(listings);
+    }
+  });
+});
+
+app.post('/rooms/similarListings', (req, res) => {
```

Comment:
Why are you opening the api to a POST request?

Reply:
I wanted to be able to trigger to the database setup from the UI. However this feature might not be very necessary, I'll can always take this down later. Thanks for pointing this @rafaelhernandez